dex/src/main/scala/com/wavesplatform/dex/cli/WavesDexCli.scala

```diff
@@ -456,28 +448,30 @@ object WavesDexCli extends ScoptImplicits {
 
     // noinspection ScalaStyle
     OParser.parse(parser, rawArgs, Args()).foreach { args =>
-      args.command match {
+      val argsOverrides = loadMatcherSettingsUnsafe(args.configPath).cli.argsOverrides
+      val updatedArgs = argsOverrides.updateArgs(args)
+      updatedArgs.command match {
         case None => println(OParser.usage(parser, RenderingMode.TwoColumns))
         case Some(command) =>
           println(s"Running '${command.name}' command")
-          AddressScheme.current = new AddressScheme { override val chainId: Byte = args.addressSchemeByte.getOrElse('T').toByte }
+          AddressScheme.current = new AddressScheme { override val chainId: Byte = updatedArgs.addressSchemeByte.getOrElse('T').toByte }
           command match {
-            case Command.GenerateAccountSeed => generateAccountSeed(args)
-            case Command.CreateAccountStorage => createAccountStorage(args)
-            case Command.CreateDocumentation => createDocumentation(args)
-            case Command.CreateApiKey => createApiKey(args)
-            case Command.CheckServer => checkServer(args)
-            case Command.RunComparison => runComparison(args)
-            case Command.MakeOrderbookSnapshots => makeSnapshots(args)
-            case Command.CheckConfigFile => checkConfig(args)
-            case Command.CleanAssets => cleanAssets(args)
+            case Command.GenerateAccountSeed => generateAccountSeed(updatedArgs)
+            case Command.CreateAccountStorage => createAccountStorage(updatedArgs)
+            case Command.CreateDocumentation => createDocumentation(updatedArgs)
+            case Command.CreateApiKey => createApiKey(updatedArgs)
+            case Command.CheckServer => checkServer(updatedArgs)
```

Comment:
We could pass loaded matcher settings from:
```scala
val argsOverrides = loadMatcherSettingsUnsafe(args.configPath).cli.argsOverrides
```

Reply:
resolved